### PR TITLE
Merge a release branch to main

### PR DESCRIPTION
**Pull request by PiotrWolinski**

### CODE REVIEW

File: app/app.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import httpx
+import json
+
+from models import InventoryRequest
+from fastapi import FastAPI, HTTPException
+
+app = FastAPI()
+
+INVENTORY_BASE_URL = "http://inventory-service"
```
Comment (gdask) [inline app/app.py]: Maybe retrieve those values from configuration file?

File: Dockerfile [hunk 2]
```diff
@@ -87,9 +87,4 @@ COPY app/ app/
 
 EXPOSE 8000
 
-RUN addgroup -S nonroot \
-    && adduser -S nonroot -G nonroot
-
-USER nonroot
```
Comment (gdask) [inline Dockerfile]: Why do we use nonroot user here?
Do we have to set a nonroot user in the other containers as well?

**Review by gdask — COMMENTED** [review]
No particular issues for this stage of development.
Ship it!

File: Dockerfile (hunk 2 above)
Comment (PiotrWolinski) [reply Dockerfile]: Hey, this PR is already closed, so please don't comment here and contact me on WhatsApp if you want to discuss anything :)

**Pull request merged**